src/plotting/plotter.py

```diff
@@ -37,16 +36,12 @@ def _create_plot(self, plot_name: str, plot_specs: Dict):
 
     def _insert_data_in_axs(self, subplot_paths_group: List, axs: np.ndarray, plot_specs: Dict):
         for i, subplot_path_group in enumerate(subplot_paths_group):
-            file_data_hanlder = DataFileHandler()
-            match file_data_hanlder.add_dataframes_from_folders_as_mean(subplot_path_group):
-                case Ok(msg):
-                    logger.info(msg)
-
-                    subplot_df = file_data_hanlder.dataframe
-                    subplot_df = DataHandler.prepare_dataframe_for_boxplot(subplot_df)
-                    self._add_subplot_df_to_axs(subplot_df, i, axs, plot_specs)
-                case Err(msg):
-                    logger.error(msg)
+            file_data_handler = DataFileHandler()
+            file_data_handler.add_dataframes_from_folders_as_mean(subplot_path_group)
```

Comment:
If this function errors out (logging the Err and then returning), doesn't it impact the continuity of this function?

Reply:
Hmm, what do you mean? This function doesn't return if it has an error. It writes the error, but keep's reading folders. If it cannot read data, it will create an empty dataframe. And if it is missing data, that will be logged, and the dataframe will be created with missing data.

Reply:
I'm wondering whether the `add_dataframes_from_folders_as_mean` function should be doing all the "if it's ok then do stuff, it it's err then log the error" work, or just return a `Result` (as in, not handling the `Err` case) and let callers handle it.

Reply:
There is a specific reason about why I did it like this. Let me elaborate:
In the configuration file, you can set up a plot like the following example:
- I want to get the metric `bandwidth` from the waku versions `v0.24`, `v0.25` and `v0.26`.
In order to display this data in the same plot, what `add_dataframes_from_folders_as_mean` will receive as a List will be `['v0.24/bandwidth', 'v0.25/bandwidth', 'v0.26/bandwidth']`.

Each of those folders will be a subplot inside the same plot. If `v0.24/bandwidth` doesn't exist for any reason, the plot will still be created with `0.v25` and `0.v26`. So returning a `Result` here might be misleading. Because if a folder is missing, it can be argued that might be `Ok` or `Err`.